### DISCUSSION TITLE
region.content doesn't work

### DISCUSSION THREAD

**Discussion by levaweb** [katspaugh/wavesurfer.js#3198] [Q&A] [answered]
Hi , I'm using the basic example below, but when I try to access region.content I get an empty object.
I can read other props like start , end - but not content.

```
wsRegions.addRegion({
    start: 9,
    end: 20,
    content: "Hello ",
    color: randomColor(),
    minLength: 1, 
}) 
```

 
```
wsRegions.on('region-clicked', (region, e) => {
    alert (region.content);  //  Here I get:  [object HTMLDivElement]     AND if I stringify it I get  {}
})  
```

Is there a way to fix this, OR to access the shadow DOM in JS ?

Many thanks

**Comment by katspaugh** [comment, marked as answer]
Try region.content.textContent

**Reply by levaweb** [reply]
Awesome! It worked

Many thanks
